stable/traefik/templates/deployment.yaml

```diff
@@ -62,7 +71,7 @@ spec:
         - containerPort: 80
         - containerPort: 443
         {{- if .Values.dashboard.enabled }}
-        - containerPort: 8080
+        - containerPort: {{ .Values.dashboard.port }}
```

Comment:
What is the reason for this particular change? This looks unrelated to the "run using a daemonset" feature. And although this may even be a common thing to allow users to configure, I'm unaware of compelling reasons for anyone to concern themselves with what specific port a process listens on _within_ a container. I would remove this change, and if you can make a compelling case for it, please make it a separate PR.